docs/synapse/userguides/storm_ref_type_specific.rstorm

```diff
@@ -79,75 +96,80 @@ Add a name to the array of names associated with an organization:
 
 Operations
 ++++++++++
 
 Lifting and Filtering
 ~~~~~~~~~~~~~~~~~~~~~
 
-Array properties can be used to lift or filter nodes based on an **exact match** of the property value using the standard equals operator:
-
+Lifting or filtering array properties using the equals ( ``=`` ) operator requires an **exact match** of the full array property value. This makes sense for forms with simple values like ``inet:ipv4=1.2.3.4``, but is often infeasible for arrays because lifting by the **full** array value requires you to know the exact values of each of the array elements as well as their exact order:
 
 .. storm-pre:: ou:org:names=("The Vertex Project", "The Vertex Project, LLC", Vertex)
 ::
 
     ou:org:names=("The Vertex Project", "The Vertex Project, LLC", Vertex)
 
-
-However, this method requires you to know in advance the exact values of each of the array elements as well as their exact order. This is often infeasible in practice.
-
-For this reason, Storm offers a special "by" syntax for lifting and filtering with ``array`` types. The syntax consists of an asterisk ( ``*`` ) preceding a set of square brackets ( ``[ ]`` ), where the square brackets contain a comparison operator and a value that can match one or more elements in the array. This allows users to match values in the array list without knowing the exact order or (in some cases, such as a prefix match) the exact value.
+For this reason, Storm offers a special "by" syntax for lifting and filtering with ``array`` types. The syntax consists of an asterisk ( ``*`` ) preceding a set of square brackets ( ``[ ]`` ), where the square brackets contain a comparison operator and a value that can match one or more elements in the array. This allows users to match one or more elements in the array similarly to how they would match individual property values.
 
 .. NOTE::
   The square brackets used to lift or filter based on values in an array should not be confused with square brackets used to add or modify nodes or properties in :ref:`edit-mode`.
 
 **Examples:**
 
-Lift the ``ou:org`` node(s) whose ``:names`` property contains the value ``vertex``:
-
+Lift the ``ou:org`` node(s) whose ``:names`` property contains a name that exactly matches ``vertex``:
 
 .. storm-pre:: ou:org:names*[=vertex]
 ::
 
     ou:org:names*[=vertex]
 
 
-Lift the x509 certificate nodes that reference the domain ``microsoft.com``:
+Lift the ``ou:org`` node(s) whose ``:names`` property contains a name that includes the string ``vertex``:
+
+.. storm-pre:: ou:org:names*[~=vertex]
```

Comment:
Do we have real examples we can show for these instead of using the storm-pre block?

Reply:
Updated "type specific" doc to include for real examples.